src/Cms/Site.php

```diff
@@ -360,11 +360,15 @@ public function mediaUrl(): string
      *
      * @param string|null $format
      * @param string|null $handler
-     * @return mixed
+     * @return int|string
```

Comment:
`Dir::modified()` returns `int`?

https://github.com/getkirby/kirby/blob/3.5.3.1/src/Toolkit/Dir.php#L257

Reply:
That type hint is incorrect as well. If a format is used, the return value is indeed `string`.

Reply:
That type hint is incorrect as well. If a format is used, the return value is indeed `string`.